app/assets/stylesheets/admin/dialog.css.scss

```diff
@@ -2,4 +2,32 @@
   p {
     line-height: 2;
   }
+
+  .alert-box{
```

Comment:
don't we have already styles for an alert box we can reuse? It feels weird to me that at the point we're at with OFN we have to come up with styles for an alert message :thinking: There must be something.